### ISSUE TITLE
"Create DOE Prototype Building" BCL measure hot water heaters malfunctioning

### ISSUE THREAD

**Issue by mldichter** [NREL/openstudio-standards#443]
Running "Create DOE Prototype Building" BCL measure with parameters
RetailStripmall
90.1-2010
ASHRAE 169-2006-2A
Not Applicable

Has each of the hot water heaters overheating over 100,000 times, for a total of over 800,000 overheatings for all the hot water heaters.

** Warning ** Plant loop exceeding upper temperature limit, PlantLoop="LGSTORE2 ZN SERVICE WATER LOOP"
**   ~~~   **   This error occurred 118841 total times;
**   ~~~   **   during Warmup 1630 times;
**   ~~~   **   during Sizing 0 times.
**   ~~~   **   Max=82.222222

EnergyPlus Completed Successfully-- 828840 Warning; 0 Severe Errors; Elapsed Time=00hr 00min 44.98sec

**Comment by mldichter** [comment]
This is occurring because of the amount of heat being put into the water from off cycle energy consumption, which is presumably whatever electronics are monitoring the tank. The off cycle heat input is larger than the off cycle heat loss to the environment, which means the water is accumulating heat. This accumulation of heat combined with little demand for hot water in the building eventually puts the water in the tank past the Maximum Temperature Limit and the water stays that way for most of the rest of the simulation. I suspect something isn't quite right about at least one of these fields:
Off Cycle Parasitic Fuel Consumption Rate {W}
Off Cycle Parasitic Heat Fraction to Tank
Off Cycle Loss Coefficient to Ambient Temperature {W/K}

This overheating of the water heaters is not limited to the Strip Mall. Many of the building types have this problem.

```
WaterHeater:Mixed,
  40gal Electricity Water Heater - 40kBtu/hr 1 1.0 Therm Eff, !- Name
  0.151416471358794,                      !- Tank Volume {m3}
  Service Water Loop Temp - 140F 1,       !- Setpoint Temperature Schedule Name
  2,                                      !- Deadband Temperature Difference {deltaC}
  82.2222222222223,                       !- Maximum Temperature Limit {C}
  Cycle,                                  !- Heater Control Type
  11722.8428068889,                       !- Heater Maximum Capacity {W}
  ,                                       !- Heater Minimum Capacity {W}
  0,                                      !- Heater Ignition Minimum Flow Rate {m3/s}
  0,                                      !- Heater Ignition Delay {s}
  Electricity,                            !- Heater Fuel Type
  1,                                      !- Heater Thermal Efficiency
  ,                                       !- Part Load Factor Curve Name
  172.999852722663,                       !- Off Cycle Parasitic Fuel Consumption Rate {W}
  Electricity,                            !- Off Cycle Parasitic Fuel Type
  0.8,                                    !- Off Cycle Parasitic Heat Fraction to Tank
  172.999852722663,                       !- On Cycle Parasitic Fuel Consumption Rate {W}
  Electricity,                            !- On Cycle Parasitic Fuel Type
  0,                                      !- On Cycle Parasitic Heat Fraction to Tank
  Zone,                                   !- Ambient Temperature Indicator
  ,                                       !- Ambient Temperature Schedule Name
  LGstore2 ZN,                            !- Ambient Temperature Zone Name
  ,                                       !- Ambient Temperature Outdoor Air Node Name
  1.205980747,                            !- Off Cycle Loss Coefficient to Ambient Temperature {W/K}
  1,                                      !- Off Cycle Loss Fraction to Zone
  1.205980747,                            !- On Cycle Loss Coefficient to Ambient Temperature {W/K}
  1,                                      !- On Cycle Loss Fraction to Zone
  ,                                       !- Peak Use Flow Rate {m3/s}
  ,                                       !- Use Flow Rate Fraction Schedule Name
  ,                                       !- Cold Water Supply Temperature Schedule Name
  40gal Electricity Water Heater - 40kBtu/hr 1 1.0 Therm Eff Supply Inlet Water Node, !- Use Side Inlet Node Name
  40gal Electricity Water Heater - 40kBtu/hr 1 1.0 Therm Eff Supply Outlet Water Node, !- Use Side Outlet Node Name
  1,                                      !- Use Side Effectiveness
  ,                                       !- Source Side Inlet Node Name
  ,                                       !- Source Side Outlet Node Name
  1,                                      !- Source Side Effectiveness
  Autosize,                               !- Use Side Design Flow Rate {m3/s}
  Autosize,                               !- Source Side Design Flow Rate {m3/s}
  1.5,                                    !- Indirect Water Heating Recovery Time {hr}
  IndirectHeatPrimarySetpoint,            !- Source Side Flow Control Mode
  ,                                       !- Indirect Alternate Setpoint Temperature Schedule Name
  General;                                !- End-Use Subcategory
```

**Comment by mldichter** [comment]
Request that this issue be moved to NREL/OpenStudio-measures issues.

**Comment by lymereJ** [comment]
Fixed by #581.
